guacamole-ext/src/main/java/org/apache/guacamole/net/auth/AbstractAuthenticatedUser.java

```diff
@@ -29,13 +34,39 @@
 public abstract class AbstractAuthenticatedUser extends AbstractIdentifiable
         implements AuthenticatedUser {
 
+    /**
+     * The logger for this class.
+     */
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractAuthenticatedUser.class);
+    
+    /**
+     * The server environment in which this Guacamole Client instance is
+     * running.
+     */
+    private final Environment environment = LocalEnvironment.getInstance();
+    
     // Prior functionality now resides within AbstractIdentifiable
 
     @Override
     public Set<String> getEffectiveUserGroups() {
         return Collections.<String>emptySet();
     }
 
+    @Override
+    public boolean isCaseSensitive() {
+        try {
+            return environment.getCaseSensitivity().caseSensitiveUsernames();
+        }
+        catch (GuacamoleException e) {
+            LOGGER.warn("Exception attempting to read the Guacamole configuration, "
+                      + "usernames will be treated as case-sensitive.", e.getMessage());
```

Comment:
`{}` placeholder for `e.getMessage()` missing here.

Reply:
Added via rebase.